LayoutTests/imported/w3c/web-platform-tests/WebIDL/ecmascript-binding/legacy-platform-object/Set.html

```diff
@@ -32,6 +32,21 @@
         assert_equals(element.attributes.foo, "foo");
     }, "must allow assigning to a named property on an object which implements a named property getter but not a setter in strict mode");
 
+    test(function() {
```

Comment:
Improved WPT test coverage here to cover the bug.

Upstream PR: https://github.com/web-platform-tests/wpt/pull/40734